api/client/samples/analogous_years/lib/final_ranks_computation.py

```diff
@@ -78,17 +78,16 @@ def time_series(client, data_series, initial_date, final_date):
     :return: A dataframe with data from the relevant dates
     """
     logger = client.get_logger()
-    entities = {'metric_id', 'item_id', 'region_id', 'source_id', 'frequency_id', 'start_date'}
+    entities = {'metric_id', 'item_id', 'region_id', 'source_id', 'frequency_id', 'start_date_bound'}
     discard = []
     for entity_type in data_series:
         if entity_type not in entities:
             discard.append(entity_type)
     for entity_type in discard:
         data_series.pop(entity_type)
     data = get_transform_data.get_data(client, **data_series)
-    consolidated_data = get_transform_data.combine_subregions(data)
```

Comment:
what is the reason for moving this call from within `get_data` to out here?

Reply:
Think of corn belt. It will have ten rows with same date, which will make resampling by nearest method more complex. Hence, I moved it upstream. Now, I do not have to use a fill function separately.